src/core/evaluator.js

```diff
@@ -387,6 +388,48 @@ class PartialEvaluator {
     return data;
   }
 
+  async fetchStandardFontData(name) {
```

Comment:
Similar to the `fetchBuiltInCMap` above, please cache the *raw* font data using a `standardFontCache` (see the `builtInCMapCache` implementation) to avoid having to repeatedly fetch the same font data over and over.